packages/comms/utils/index.ts

```diff
@@ -38,6 +39,11 @@ const outDir = args?.outDir ? args?.outDir : "./temp/wsdl";
 
 const ignoredWords = ["targetNSAlias", "targetNamespace"];
 
+const tsFmtOpts = {
+    replace: true, verify: false, tsconfig: true, tsconfigFile: null, tslint: true, tslintFile: null,
+    editorconfig: true, vscode: true, vscodeFile: null, tsfmt: false, tsfmtFile: null
+};
+
```

Comment:
Do these options match the VSCode formatting?  IOW if I open a generated file and re format it in VSCode will it change?

Reply:
What does `replace: true` do?

Reply:
They do match the current VSCode formatting, I think perhaps only coincidentally because there's not anything defined that overrides the defaults styles.

The replace option decides whether the script should overwrite the file in place.